esx_service/utils/vmdk_utils.py

```diff
@@ -248,6 +248,15 @@ def get_vm_uuid_by_name(vm_name):
     except:
         return None
 
+def get_vm_name_by_uuid(vm_uuid):
+    """ Returns vm_name for given vm_uuid, or None """
+    si = vmdk_ops.get_si()
+    try:
+        vm = [d for d in si.content.rootFolder.childEntity[0].vmFolder.childEntity if d.config.uuid == vm_uuid]
+        return vm[0].config.name
```

Comment:
Then how do we handle this? Assume we already has an VM named "vm1", then I try to create another vm also named "vm1". Then how can I tell which VM is the one I just created?